src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
```

Comment:
- [ ] Please kindly remove commented code from your code base as it makes your code dirty and less readable.